mobile/src/main/java/com/caverock/androidsvg/IntegerParser.java

```diff
@@ -0,0 +1,177 @@
+/*

```

Comment:
Hi, not sure what you mean. The problem is that the current version of androidsvg is missing a pending bugfix (https://github.com/BigBadaboom/androidsvg/pull/99) which is needed to work in habdroid. The copy I added is patched.
I guess it is possible to create a separate gradle project which contains the patched version, but I tried to follow the pattern used for de.duenndns.ssl.